albumentations/augmentations/transforms.py

```diff
@@ -3378,3 +3467,87 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class AugMix(ImageOnlyTransform):
+    """Augmentation from "AugMix: A Simple Data Processing Method to Improve Robustness and Uncertainty"
+    Please note that this augmentation performs normalization internally, so resulting image will be float type.
+
+    Args:
+        alpha (float): Probability coefficient for Beta and Dirichlet distributions. Default: (1.0).
+        width (int): Width of augmentation chain. Default: 3.
+        depth (int, tuple of ints): Depth of augmentation chain. If single int will be used provided number.
+            If tuple of depth will be generated in range `[depth[0], depth[1])`. Default: (3).
+        transforms (list of albumentation transforms): List of transforms from which augmentation will be sampled
+            on each step of AugMix procedure.
+        mean (float, list of floats, tuple of float): mean values for normalization. Default: (0.485, 0.456, 0.406).
+        std (float, list of floats, tuple of floats): std values for normalization. Default: (0.229, 0.224, 0.225).
+    Targets:
+        image
+
+    Image types:
+        uint8, float32 3-channel images only
+
+    Credit:
+        https://arxiv.org/pdf/1912.02781.pdf
+        https://github.com/google-research/augmix
+    """
+
+    def __init__(
+        self,
+        alpha=1.0,
+        width=3,
+        depth=3,
+        transforms=None,
+        mean=(0.485, 0.456, 0.406),
+        std=(0.229, 0.224, 0.225),
+        always_apply=False,
+        p=0.5,
+    ):
+        super().__init__(always_apply=always_apply, p=p)
+        self.alpha = alpha
+        self.width = width
+        self.depth = to_tuple(depth, low=1)
+        self.transforms = transforms
+
+        if self.transforms is None:
+            self.transforms = [
+                Autocontrast(),
```

Comment:
All transforms need to have `p=1`, such that they are always executed when selected by `Augmix`.

Maybe add a comment and change default values ?